helix-core/src/main/java/org/apache/helix/cloud/event/HelixEventHandlingUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.helix.cloud.event;
+
+import org.apache.helix.BaseDataAccessor;
+
+
+public class HelixEventHandlingUtil {
+
+  public static boolean enableInstanceForCloudEvent(String clusterName, String instanceName, boolean isEnable, long timeStamp,
+      BaseDataAccessor dataAccessor) {
+    return true;
+  }
+
+  public static boolean IsInstanceDisabledForCloudEvent(String clusterName, String instanceName,
+      BaseDataAccessor dataAccessor) {
+    return true;
+  }
+
+  public static Long getInstanceDisabledTimestamp(String clusterName, String instanceName,
```

Comment:
Is this timestamp for cloud event disable or for the general disable?

Reply:
Is intend to be the timestamp for cloud event. I updated the signature to take a message. It will be less ambiguous.